### ISSUE TITLE
채팅

### ISSUE THREAD

**Issue by burnyst** [burnyst/team-intro#4]
채팅방이에요

**Comment by burnyst** [comment]
soylent 업무 분담해요

**Comment by isyoo1213** [comment]
우선 팀장님 develop 브랜치로 풀리 보내놨습니다.

혹시 강사님꼐서 다른 과제 내주신 것은 기한이 언제까지라고 말씀해주셨나요..?

**Comment by burnyst** [comment]
리퀘스트 머지 했습니다
과제 내용은 공지로 나갔네요 ^^

**Comment by ShadowOfShadow** [comment]
저도 올렸습니다

**Comment by burnyst** [comment]
저 빼고 다들 잘 하셨어요 ㅋㅋ
이제 https://soylent.com/ 클론 페이지 만들어요

**Comment by burnyst** [comment]
충돌 처리를 쉽게 하기 위해 파트를 나눠서 작업해야 되요

**Comment by burnyst** [comment]
파트 정하고 이슈 만들어서 협업 진행하시죠

**Comment by ShadowOfShadow** [comment]
아 저 과제를 지금 봐가지고 좀 큰 사이트네요

**Comment by ShadowOfShadow** [comment]
각자 그 파트별로 [ ] 추가해서 이슈 만들면 될거같은데 그 전에 파트 정하면 될거같습니다

**Comment by ShadowOfShadow** [comment]
개발자도구 열어서 div id 별로 각자 하고싶은 파트 맡으시면될거같아요

**Comment by ShadowOfShadow** [comment]
제가 div id = shopify-section-16243360044c1367c3 이거 함 해볼게요 잘될지 모르겠지만 ㅋㅋ

**Comment by ShadowOfShadow** [comment]
head 부분 맡으실분 있어야될거같아요

**Comment by ShadowOfShadow** [comment]
banner 부분이랑 header-container 이 2개를 하실분이랑
DRINKS POWDERS BARS MERCH 이부분의 head 하실분을 각자 정하면 될거같은데 어떻게 생각하시나요?

**Comment by burnyst** [comment]
인성님 채팅 걸었으니까 기다려보죠

**Comment by isyoo1213** [comment]
제가 html을 거의 안해봐서..  학습하면서 진행해야할 것 같은데 괜찮으실까요..?

**Comment by burnyst** [comment]
상관없어요 단지 깃헙 가지고 협업하는거 연습하는 거니까요

**Comment by ShadowOfShadow** [comment]
네 저도 잘 못해요.. 장기적으로 하면 될거 같습니다 기한이 무제한인거보니깐 잊어먹을떄마다 이런 사이클을 계속 하면 실수를 줄일 수 있는거니깐요 ㅎㅎ

**Comment by burnyst** [comment]
제생각 은 헤더(메뉴 포함)랑 푸터랑 한분 하시고

**Comment by burnyst** [comment]
Perks of Subscribing까지 한명, 나머지 한명 이렇게 나누는 건 어떨까요?

**Comment by isyoo1213** [comment]
네 좋습니다!

**Comment by burnyst** [comment]
1. Find Your Perfect Soylent ~ Perks of Subscribing
2. Good For You, Good for the Planet. ~ Follow Us @Soylent
3. 나머지

**Comment by burnyst** [comment]
CSS가 걸리네요

**Comment by ShadowOfShadow** [comment]
css는 강사분에게 물어보거나 아니면 그 개발자도구에서 network에서 css 필터로 보니깐
그거 그냥 복붙해도 될거같아요

**Comment by burnyst** [comment]
충돌때문에요

**Comment by burnyst** [comment]
세파트 공통 부분 스타일이 많을 텐데

**Comment by burnyst** [comment]
중복되도 스타일 각자 하죠

**Comment by ShadowOfShadow** [comment]
팀장님이 저희 프로젝트 넣을 폴더 만들어주실 수 있으신가요?

**Comment by ShadowOfShadow** [comment]
그 거기다가 css 다운받아서 pull request 해놓겠습니다.

**Comment by burnyst** [comment]
네 그러면 css 다운 받은 거 그대로 쓰죠

**Comment by ShadowOfShadow** [comment]
전부다 다운 받는건 아니여서 부족한건 그때 부딪혀가면서 해야될거같습니다

**Comment by ShadowOfShadow** [comment]
그래도 최대한 공통된거 다 받을 수 있을거같아요

**Comment by burnyst** [comment]
만들었어요

**Comment by burnyst** [comment]
하고 싶은 파트 번호 선택해 주세요
남은 부분은 제가 할게요

**Comment by burnyst** [comment]
오늘 중으로 선택 안 하신 분은 제가 어려운 부분 배정해 드립니다 ㅋ

**Comment by isyoo1213** [comment]
1번 헤딩하면서 해보겠습니다!

**Comment by ShadowOfShadow** [comment]
제가 2번 하면 될거같습니다 헤딩 해봐야될거같은데 일단 지금 버그때문이 이유는 모른체로 css push가 안되서요 강사님한테 질문했습니다

**Comment by ShadowOfShadow** [comment]
해결됬습니다 ㅋㅋ

**Comment by ShadowOfShadow** [comment]
css pull request 보내놨습니다

**Comment by burnyst** [comment]
제가 3번 할게요 ㅋ
index.html에 각자 작업할 영역 표시한 다음에 알려드릴게요

**Comment by ShadowOfShadow** [comment]
네 그 css 폴더를 나중에는 soylent 안에 옮겨야겠네요 제가 처음부터 그랬어야 됬는데 실수했네요 ㅋㅋ

**Comment by burnyst** [comment]
네 그렇게 하세요. 저도 검토했는데 놓쳤네요. soylent 안에 한 줄 알았어요 ㅋ
index.html 수정했어요. 각자 깃허브 아이디 주석 아래에 작업하시고 머지 하시면 됩니다.

**Comment by ShadowOfShadow** [comment]
넹 fetch -> merge 했습니다. 작업은 좀 천천히 시작해도될까요? 아직 html하고 javascript가 익숙하지 않아서 다시 복습하고 있습니다.

**Comment by burnyst** [comment]
자주 커밋하는게 포인트인 것 같아요.
하루에 한번 정도만 조금 수정하고 커밋하고 풀 리퀘스트 하면 될 것 같아요

**Comment by burnyst** [comment]
바디 태그 속성 수정하고 올렸어요

**Comment by ShadowOfShadow** [comment]
네 받아놓겠습니다!

**Comment by burnyst** [comment]
배너 추가하고 올렸어요

**Comment by ShadowOfShadow** [comment]
네 고생하셨습니다!

**Comment by burnyst** [comment]
MainContent 태그를 main으로 변경했어요

**Comment by isyoo1213** [comment]
git flow 연습겸 첫 타이틀 출력하는 코드 추가해서 풀리 보내봤습니다!
아직 issue등록을 하지 않아서 approve하지 않으셔도 좋습니다!
